adagrams/game.py

```diff
@@ -1,11 +1,147 @@
+import random
+import copy
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+# Helper Functions --- Start ------
+
+def create_word_list(word):
+    word_list =[]
+    for each in word:
+        word_list.append(each.upper())
+    return word_list
+
+# Helper Functions --- End --------
+
 def draw_letters():
-    pass
+    hand = []
+    while len(hand) < 10 or not hand:
+        letter = random.choice(list(LETTER_POOL))
+        if hand.count(letter) < LETTER_POOL[letter]:
+            hand.append(letter)
+
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word_list = create_word_list(word)
+    letter_bank_copy = copy.deepcopy(letter_bank)
+
+    # for each in word:
+    #     word_list.append(each.upper())
```

Comment:
Since we are not using this code, let's get rid of it for our final submission.

```suggestion
    # for each in word:
    #     word_list.append(each.upper())
```